src/rootNormLhalf.jl

```diff
@@ -29,11 +29,11 @@ function (f::RootNormLhalf)(x::AbstractArray{T}) where {T <: Real}
 end
 
 function prox!(y::AbstractArray{T}, f::RootNormLhalf, x::AbstractArray{T}, gamma::Real=1) where {T <: Real}
-  γλ = 2 * gamma * f.lambda
-  ϕ(z) = acos(γλ / 8 * (abs(z) /3 )^(-3/2))
+  γλ = gamma * f.lambda
+  ϕ(z) = acos(γλ / 4 * (abs(z) /3 )^(-3/2))
   ysum = zero(T)
   for i in eachindex(x)
-    if abs(x[i]) <= 54^(1/3) * (γλ^(2/3)) / 4
+    if abs(x[i]) <= 54^(1/3) * ((2γλ)^(2/3)) / 4
```

Comment:
If we are really being stingy here, would this add a another multiplication every loop here?

Reply:
Yes but the entire rhs is reevaluated at each pass. That's a good comment. I'll factor it out.